ajax/libs/pixi-filters/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "pixi-filters",
+  "version": "1.0.6",
+  "description": "Optional display filters to work with Pixi.js",
+  "author": "Mat Groves",
+  "filename": "filters.min.js",
+  "homepage": "http://goodboydigital.com/",
```

Comment:
@bigtimebuddy is this url still its hompage? Or maybe we should use `https://pixijs.github.io/pixi-filters/docs/` instead?